test.flakiness.sh

```diff
@@ -18,6 +18,10 @@ if [[ "${LAST_COMMIT_MESSAGE}" == *"#extraflaky"* ]]; then
 fi
 
 if [[ $1 == "NIGHTLY" ]]; then
+    curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
```

Comment:
Please move to a separate pr. This is not related to the testnet.

Reply:
Moved already and also merged to master. Will be deleted from here.